UIKitMovie/UIKitMovie/Helpers/NetworkService.swift

```diff
@@ -0,0 +1,51 @@
+// NetworkService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Alamofire
+import Foundation
+import SwiftyJSON
+
+/// Сетевой сервис
+final class NetworkService: NetworkServiceProtocol {
+    // MARK: - Private Constants
+
+    private enum Constants {
+        static let baseURL = "https://api.themoviedb.org/3/movie/"
+        static let APIKey = "api_key"
+        static let APIKeyValue = "b6abe1b1835ab9f0603050760032a03a"
+        static let languageKey = "language"
+        static let languageValue = "ru-RU"
+        static let pageKey = "page"
+        static let pageValue = "1"
+        static let imageStartName = "https://image.tmdb.org/t/p/w500"
+    }
+
+    // MARK: - Public Methods
+
+    func fetchMovies(method: MethodType, completion: @escaping (Result<JSON, Error>) -> Void) {
+        let parameters: Parameters = [
+            Constants.APIKey: Constants.APIKeyValue,
+            Constants.languageKey: Constants.languageValue,
+            Constants.pageKey: Constants.pageValue
+        ]
+        let path = "\(Constants.baseURL)\(method.method)"
+        AF.request(path, parameters: parameters).responseData { response in
+            switch response.result {
+            case let .success(value):
+                let json = JSON(value)
+                completion(.success(json))
+            case let .failure(error):
+                completion(.failure(error))
+            }
+        }
+    }
+
+    func fetchImage(_ url: String, _ completion: @escaping (Data?) -> Void) {
+        AF.request("\(Constants.imageStartName)\(url)").response { response in
+            guard response.data != nil else { return }
+            DispatchQueue.main.async {
+                completion(response.data)
+            }
```

Comment:
```suggestion
                completion(response.data)
```
 разве после выполнения запроса нас не перекидывают на main?

Reply:
и ошибку тоже надо обрабатывать